### ISSUE TITLE
worked sort of , deleted & reloaded , now it's gone

### ISSUE THREAD

**Issue by atechnical** [kevinvincent/ha-wyzesense#60]
[<script src="https://pastebin.com/embed_js/XnvsSsU2"></script>](url)

I loaded it,two sensors appeared in my overview but the scan service never became available. I deleted the integration, tried loading the integration by loading the custom repository in hassio web gui and the manual installation but now it won't even appear in the HACS Community Store

**Comment by atechnical** [comment]
Forgot to mention I'm running  hassio on raspberry pi 3b+

**Comment by gdreelin** [comment]
Yeah running this on Proxmox through a NUC and recent update made them and the services all disappear. I did the auto change like it stated in the Pull Request is worked for a moment and then nothing. The services are not showing up to re-add the devices.

**Comment by atechnical** [comment]
There was a HACS update available that I downloaded then I reinstalled via github and then from the icon available in the integration in the HACS store . It's working again

**Comment by gdreelin** [comment]
> 
> 
> There was a HACS update available that I downloaded then I reinstalled via github and then from the icon available in the integration in the HACS store . It's working again

So just manually install it again from the Github and then when it appears in HACS just do a reinstall? I thought I had tried that but will do it again.

**Comment by carver7** [comment]
Updating from HACS v0.16.3 to 0.17.4 caused:

1.  all but 2 or 3 contact switches disappeared from hassio (down from about 12).
2. In HA UI:  in section after clicking Community button in sidebar: a message in a box was displayed above the Integrations section (maybe also under the Community Settings) saying something like 'Wyzesense component not loaded in Home Assistant'

Restoring a HA Snapshot brought all the all contact sensors back, so I'm still running HACS 0.16.3.
@gdreelin : Are you still having problems after manual re-install ?

(My setup: hassio alt generic linux install in Docker on RPi 4; Home Assistant v 0.101.3)

**Comment by gdreelin** [comment]
> 
> 
> Updating from HACS v0.16.3 to 0.17.4 caused:
> 
>     1. all but 2 or 3 contact switches disappeared from hassio (down from about 12).
> 
>     2. In HA UI:  in section after clicking Community button in sidebar: a message in a box was displayed above the Integrations section (maybe also under the Community Settings) saying something like 'Wyzesense component not loaded in Home Assistant'
> 
> 
> Restoring a HA Snapshot brought all the all contact sensors back, so I'm still running HACS 0.16.3.
> @gdreelin : Are you still having problems after manual re-install ?
> 
> (My setup: hassio alt generic linux install in Docker on RPi 4; Home Assistant v 0.101.3)

I had to back up to few days ago snapshot. Once I did that the upgrade to 0.17.4 HACS seemed to work to bring back the sensors but the services to scan no longer show up. So backed it back down again to that snapshot which had 0.16.3 HACS and same issue the scanning ability is gone but it no longer shows red in HACS. I did try to manually add them all again but since the scan service is not being found it would not let me do that so I just redid the snapshot down one. One thing I noticed too is that the sensors are showing in Entities but they say they are unavailable not sure there.

**Comment by gdreelin** [comment]
Ok after completely rebuilding my Hassio again (separate issue) I reinstalled the Wyzesense again, set it to "auto" first but no joy on getting the services to show. I changed it to "/dev/hidraw0" and to "/dev/hidraw1" which are the only two showing up. Nothing! It just seems like this addon died after they went to version 100 HA.

**Comment by atechnical** [comment]
Yeah I've given up. I moved from a raspberry pi 3b+to a virtual Linux
machine and I can't get it to work at all anymore. Not sure if that's
because it's a virtual machine or it's dead in some other way. I'll wait a
while before I try and pick it up again maybe things will get better by
then.

**Comment by kevinvincent** [comment]
Hey, could you try updating to the latest version `0.0.7` and let me know if it works? It should fix a very common stability issue.

**Comment by atechnical** [comment]
Thanks for getting back to me.
Unfortunately I moved from a Raspberry PI to a virtual machine.
I can't replicate the error or confirm the update fixed it.
The good news is it's running like a friggin champ on my virtual machine.
I have motion and door sensors all over the house .
My wife and kids hate me think I'm the NSA...it's GREEEAT!!

You are da man.
